### PR TITLE
ui: Set Link Color when setting theme

### PR DESCRIPTION
**Pull request by Docteh**
Long story short, QT doesn't allow the link colors to be set via their stylesheets.

There are two ways to work with this, specify the color manually for every link (See the About dialog) The other way is to change the default palette.

IsDarkTheme is copy/pasted from src/yuzu/debugger/wait_tree.cpp

I considered setting the link color to be the same light blue as in the license dialog, but with a bit of testing I found that it wasn't a good fit. I lightened it for the dark theme, and darkened it for the light theme. 

Before
image

After
image

### CODE REVIEW

File: src/yuzu/main.cpp [hunk 1]
```diff
@@ -3652,6 +3652,18 @@ void GMainWindow::UpdateUITheme() {
         setStyleSheet({});
     }
 
+    QPalette newPal(qApp->palette());
+    if (UISettings::IsDarkTheme()) {
+        newPal.setColor(QPalette::Text, QColor(255,255,255,255));
+        newPal.setColor(QPalette::Link, QColor(0,190,255,255));
+        //newPal.setColor(QPalette::LinkVisited, QColor(0,190,255,255));
```
Comment (lioncash) [inline src/yuzu/main.cpp:3659]: Generally we prefer to not keep commented out code around, ditto for the else case

File: src/yuzu/main.cpp [hunk 2]
```diff
@@ -3652,6 +3652,18 @@ void GMainWindow::UpdateUITheme() {
         setStyleSheet({});
     }
 
+    QPalette newPal(qApp->palette());
```
Comment (lioncash) [inline src/yuzu/main.cpp:3655]:
```suggestion
    QPalette new_pal(qApp->palette());
```

**Comment by Docteh** [comment]
hi, thanks for the review, I will check into it, and the clang formatting thing in a bit

**Comment by german77** [comment]
Changing between light and dark theme will render the game list incorrectly. This doesn't happen on mainline
image

**Comment by Docteh** [comment]
That is a very interesting interaction between touching QPalette::Text and the themes. 

On the plus side just touching QPalette::Link is working going from light to dark to light again.

**Review by german77 — APPROVED** [review]

**Review by bunnei — APPROVED** [review]

**Comment by goldenx86** [comment]
Hi, can I mention your PRs in the next progress report?

**Pull request merged**